ocean_lib/models/test/test_datatoken_enterprise.py

```diff
@@ -114,18 +90,10 @@ def test_buy_from_dispenser_and_order(
     _ = DT.buy_from_dispenser_and_order(
```

Comment:
Right now, the call to this method is returning a "_". But we should show what it returns. Is it order_tx_id? More?

Also, after this call, I think there should probably be some basic sanity test that the order_tx_id is valid.

Reply:
This is now 
```python
_ = DT.buy_from_exchange_and_order(...)
```

But the comment above still needs to be resolved.